CaptainJack.driver/Contents/MacOS/CaptainJack

```diff
@@ -1 +0,0 @@
-../../../build/src/captain-jack
```

Comment:
Nevermind, I see what happened. `install` doesn't like symlinks. Hmm.. okay. Disregard this.